### PR TITLE
Injection http.Client for GAE/Go

### PR DESCRIPTION
**Pull request by eiel**
I was able to use go-chatwork in GAE

### CODE REVIEW

**Comment by griffin-stewie** [comment]
@eiel 

LGTM your PR :+1: , but Can you fix naming fields `HttpClient` to `HTTPClient` for naming convention.

Please see bellow.

screenshots_atom_warnning

**Comment by eiel** [comment]
Thank you.I fixed to `HTTPClient`

File: client.go [hunk 1]
```diff
@@ -76,7 +77,9 @@ func (c *Client) parseBody(resp *http.Response) []byte {
 }
 
 func (c *Client) execute(method, endpoint string, params map[string]string) []byte {
-	httpClient := &http.Client{}
+	if c.HttpClient == nil {
```
Comment (griffin-stewie) [inline client.go]: @eiel 
You missed change `HttpClient` to `HTTPClient`.

File: client.go [hunk 2]
```diff
@@ -76,7 +77,9 @@ func (c *Client) parseBody(resp *http.Response) []byte {
...
+		c.HttpClient = &http.Client{}
```
Comment (griffin-stewie) [inline client.go]: @eiel 
You missed change `HttpClient` to `HTTPClient`.

File: client.go [hunk 3]
```diff
@@ -95,7 +98,7 @@ func (c *Client) execute(method, endpoint string, params map[string]string) []by
 
 	req.Header.Add("X-ChatWorkToken", c.APIKey)
 
-	resp, err := httpClient.Do(req)
+	resp, err := c.HttpClient.Do(req)
```
Comment (griffin-stewie) [inline client.go]: @eiel 
You missed change `HttpClient` to `HTTPClient`.

File: README.md [hunk 4]
```diff
@@ -33,4 +33,13 @@ func main() {
 }
 ```
 
+### for GoogleAppEngine/Go
+
+```go
+c := appengine.NewContext(r)
+client := urlfetch.Client(c)
+chatwork := chatwork.NewClient(`api-key`)
+chatwork.HttpClient = client
```
Comment (griffin-stewie) [inline README.md]: @eiel 
You missed change `HttpClient` to `HTTPClient`.

**Comment by eiel** [comment]
oh..I'm sorry.I fixed.

**Comment by griffin-stewie** [comment]
[:+1:](http://lgtm.in/u/8ksdQQzRp)

**Pull request merged**